apps/pigment-css-next-app/src/app/grid/demo3.tsx

```diff
@@ -0,0 +1,43 @@
+'use client';
```

Comment:
This is the only demo from the docs demos that is a client component, all others run as RSC 🚀 

The reason this has to be a client component is that it uses state to interactively show spacing changes.

Reply:
AWESOME!!! ❤️